### ISSUE TITLE
Content missing after heading - SiteImprove - Vertical Video - uiowa.edu

### ISSUE THREAD

**Issue by bspeare** [uiowa/uiowa#8515]
# Problem 

Vertical video on uiowa.edu is getting flagged in Siteimprove for content missing after heading.

The vertical video used to have content associated with the heading.  Now it longer does and we should remove the heading tag on the vertical video overlay. 

This is a "S" Best Practice issue in Siteimprove. 

Example - https://my2.siteimprove.com/Inspector/995976/A11y/Page?pageId=30184895912&impmd=264ED37B65FA6186129334A508391970#/sia-r78/failed/VWdGUGc0MGc=;MjQ5NmNiNTE=

# Solution

- ?
